docs/blog/announcing-cientos-3-2-0.md

```diff
@@ -0,0 +1,108 @@
+---
+sidebar: false
+---
+
+# Announcing cientos v3.2.0 🎉
+
+There is a lot going on in the TresJs Ecosystem.
+
+But for now let's hijack this space to announce what's happening in your favorite helper library for creating 3D experience with TresJs, the Cientos package! 😊
+
+- **Cientos v3.2.0**
+
+<iframe src="https://giphy.com/embed/vmon3eAOp1WfK" width="480" height="357" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/celebration-excited-loki-vmon3eAOp1WfK">via GIPHY</a></p>
+
+## Remove TweakPane from cientos 😱
+
+Ok ok, not really, we haven't dropped TweakPane from Cientos yet.
+
+In Cientos, we're constantly working to improve your development experience, and we have found that the tweakPane EssentialsPlugin was taking up HALF of the Cientos bundle size, so we have taken the decision to drop it from Cientos and add it as a peerDependency.
```

Comment:
This is not entirely true, what was taking half of the bundle size was `tweakpane + plugin-essentials` at least 300kb (31%). That why now `tweakpane` is a peer dependency and we drop support for essential plugins

<img width="1479" alt="260297189-9062618d-f7ac-4aad-9b20-9631e4bd2623 (1)" src="https://github.com/Tresjs/tres/assets/4699008/2d55d3bb-c9a4-49d5-a4fa-e75ff055c740">